Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -491,8 +487,10 @@ void UpdateProgressBarVisibility(bool isBusy)
 		{
 			if (!Forms.SupportsProgress)
 				return;
-
+#pragma warning disable 612, 618
```

Comment:
should we add a comment identifying whether we will revisit this disabled warning in the future, like we did with the other warnings?